### ISSUE TITLE
op selection not working in dagit (0.13.6)

### ISSUE THREAD

**Issue by amarrella** [dagster-io/dagster#5690]
## Summary

It looks like you can't execute a job and select an op correctly in dagit. When you try to select the op and specify the input it fails with the following error:
```
dagster.core.errors.DagsterInvalidConfigError: Error in config for job Error 1: Received unexpected config entry "inputs" at path...
```

## Reproduction

```python
import dagster

@dagster.op
def op1() -> str:
    return "hello"

@dagster.op
def op2(_, input_str: str) -> str:
    return input_str

@dagster.job
def job():
    op2(op1())
```

Run dagit with this job and try to run `op2` by providing the input.

## Dagit UI/UX Issue Screenshots

<img width="385" alt="Screen Shot 2021-11-17 at 16 01 00" src="https://user-images.githubusercontent.com/1425877/142225073-1d88dd7e-4d8a-4f41-8561-61942db0fb2d.png">
<img width="393" alt="Screen Shot 2021-11-17 at 16 00 52" src="https://user-images.githubusercontent.com/1425877/142225077-a35c9f77-21f5-40d6-9f08-74fd1823782e.png">
<img width="1351" alt="Screen Shot 2021-11-17 at 16 00 45" src="https://user-images.githubusercontent.com/1425877/142225080-f410560e-90f0-480a-95a0-b4915230ae08.png">

## Additional Info about Your Environment

Tested with dagster 0.13.6

---
#### Message from the maintainers:
Impacted by this bug? Give it a 👍. We factor engagement into prioritization.

**Comment by sryza** [comment]
I believe @yuhan is working on addressing this

**Comment by yuhan** [comment]
https://github.com/dagster-io/dagster/pull/5618 should fix it. It was released in `0.13.8`
